TractorMod/Framework/Attachments/ScytheAttachment.cs

```diff
@@ -46,27 +84,46 @@ public override bool Apply(Vector2 tile, SObject tileObj, TerrainFeature tileFea
                 if (dirt.crop == null)
                     return false;
 
-                if (dirt.crop.dead)
+                if (dirt.crop.dead && this.ClearDeadCrops)
                 {
                     this.UseToolOnTile(new Pickaxe(), tile); // clear dead crop
                     return true;
                 }
+
+                List<int> flowers = new List<int> { 376, 591, 593, 595, 597, 421 };
```

Comment:
Let's take out the flower feature for now, and create a separate pull request for it. I'd want to change how we get these IDs (maybe add a `data.json` to support players with custom flowers). For this pull request we can use `this.HarvestCrops` for flowers too.

Reply:
Good, that one was tricky and I wasn't really happy with the results. I guess there isn't a flag anywhere to check if a plant is a flower or not, which would be helpful.